modules/p2/ding_list/ding_list_installer/ding_list_installer.pages_default.inc

```diff
@@ -8,6 +8,105 @@
  * Implements hook_default_page_manager_pages().
  */
 function ding_list_installer_default_page_manager_pages() {
+  $page = new stdClass();
+  $page->disabled = FALSE; /* Edit this to true to make a default page disabled initially */
+  $page->api_version = 1;
+  $page->name = 'ding_list_reset_user';
+  $page->task = 'page';
+  $page->admin_title = 'Ding list reset user';
+  $page->admin_description = '';
+  $page->path = 'user/%user/reset-lists';
+  $page->access = array();
```

Comment:
This seems to warrant some form of access control. As I read the code in the current state then I would be able to reset the lists for any user if I know their user id. That is not good.

There might be permissions for deleting own and others lists further down the line but just being able to see the form here, might be enough to scare some people.